### DISCUSSION TITLE
Viomi forgets map / starts a fresh one on every clean

### DISCUSSION THREAD

**Discussion by rumpeltux** [Hypfer/Valetudo#744] [Q&A / Support]
I recently ran into the issue that whenever the Robot starts, it forgets the current map and starts a new one.
Obviously functions such as spot- or room cleaning don’t work.

After debugging for a while, I figured that a `{"method":"set_remember","params":[1]}` fixes it. So there’s actually a setting to remember the map. I didn’t toggle it, so I’m unsure how the Robot ended up in the state, but I’m glad I was able to fix it :)
Leaving this here in case others run into the same issue.

**Comment by Hypfer** [comment]
Sounds like a new `ViomiPersistentMapControlCapability` to me :)
